packages/coinstac-client-core/src/sub-api/computation-service.js

```diff
@@ -173,12 +173,65 @@ class ComputationService extends ModelService {
   }
 
   /**
+   * Allows the client to join a run for which they have no started document for.
+   * @param  {string} options.consortiumId Consortium the runId is from
+   * @param  {string} options.projectId    project to run on
+   * @param  {string} options.runId
+   * @return {Promise}
+   */
+  joinSlavedRun({ consortiumId, projectId, runId }) {
```

Comment:
This function body closely matches `ComputationService#doTriggerRunner`. DRY it out?